### PR TITLE
Tech Writer Feedback Added to Migration Guide

### PR DESCRIPTION
**Pull request by blulas**
This PR includes additional updates based on reviews by engineering and the tech writers.

### CODE REVIEW

**Review by veena-ux — COMMENTED** [review]
Some minor changes, spellings and word usages.

**Comment by blulas** [comment]
I don’t know how these files ended up in the pull request, but they are not valid to the migration guide and I have not made any changes to these files nor will I be make any changes to those files
________________________________

File: installation/docker-compose.adoc [hunk 1]
```diff
@@ -1,67 +1,29 @@
 = Docker Compose
 include::../partials/attributes.adoc[]
 
-{CANVAS} and its related services can be run on a single machine using `docker-compose`.
-We can access the docker-compose file from the https://github.com/kiegroup/kie-tools/tree/main/packages/kie-sandbox-distribution[distribution package].
+{CANVAS} and its related services can be run on a single machine using `Docker Compose for BAMOE Canvas`.
+We can access the docker-compose file from the https://www.ibm.com/support/pages/node/7001673[Support and Download Page].
 
-== Build
+== Prerequistes
```
[the remark this answers is not in the document]
Comment (blulas) [reply installation/docker-compose.adoc:7]: I changed this, but not sure why this is part of our PR for the migration guide.

File: migration-guide/drl.adoc [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+
+[source.java]
+----
+...
+then
+  $f.setWrapperLong(10L);
+end
+----
+
+Generics return type resolution, assuming you have classes below:
```
[the remark this answers is not in the document]
Comment (blulas) [reply migration-guide/drl.adoc:312]: we are holding off on the DRL section and re-assigning this to engineering for our next release.

File: installation/docker-compose.adoc [hunk 3]
```diff
@@ -1,67 +1,29 @@
...
 
-Before building the image, please set the following environment variable like so:
```
[the remark this answers is not in the document]
Comment (LightGuard) [reply installation/docker-compose.adoc:9]: This is not part of the migration guide and should not be considered for this PR.

File: installation/docker-compose.adoc [hunk 4]
```diff
@@ -1,67 +1,29 @@
...
+You will need Docker and the Compose plugin in order to run the file. Refer to their web page for installation.
```
[the remark this answers is not in the document]
Comment (LightGuard) [reply installation/docker-compose.adoc:9]: This is not part of the migration guide and should not be considered for this PR.

File: installation/docker-compose.adoc [hunk 5]
```diff
@@ -1,67 +1,29 @@
...
-export KIE_SANDBOX_DISTRIBUTION__gitCorsProxyImageAccount=<gitCorsProxyImageAccount>
-export KIE_SANDBOX_DISTRIBUTION__gitCorsProxyImageName=<gitCorsProxyImageName>
-export KIE_SANDBOX_DISTRIBUTION__gitCorsProxyImageTag=<gitCorsProxyImageTag>
-export KIE_SANDBOX_DISTRIBUTION__gitCorsProxyPort=<gitCorsProxyPort>
-----
 
 == Run
 
-Run {CANVAS} with docker compose and current environment variables using pnpm:
+To Run {CANVAS} with docker compose follow these steps:
```
[the remark this answers is not in the document]
Comment (LightGuard) [reply installation/docker-compose.adoc:17]: This is not part of the migration guide and should not be considered for this PR.

**Comment by LightGuard** [comment]
Closing this PR, @blulas will create a new PR with a clean history.

**Pull request closed without merging**